SU-HSDO/suhumsci
# SHS-5844: The Hover state of links on global banners doesn't meet contrast

# READY FOR REVIEW

## Summary
- Fix contrast of hover links in global message component.

## Need Review By (Date)
10/09

## Urgency
medium

## Steps to Test
1. Visit `/admin/config/system/global-message`
2. If disabled, enable the global message. Set the "Message type" to any value (all will be tested) and add some test content if needed. Make sure to add an action link.
3. Visit any page on the site and confirm that the default and hover state of any link in the global message match the [Figma designs](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=5776-11&t=1gSOIVS9BFQrXti8-1).
4. Go back to the global message edit form and try all other message types. Repeat the previous test.

**Note:** traditional and colorful share the same global message styles, you can test on any site and the results should be the same.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208522596918922
  - https://app.asana.com/0/0/1208439121950272
